### ISSUE TITLE
Codepoints file

### ISSUE THREAD

**Issue by scriptPilot** [nolimits4web/Framework7-Icons#7]
Hi Vladimir,

There are issues with older browsers / Android versions to display ligatures (see https://github.com/google/material-design-lite/issues/1375).

For Material icons, there is a mapping file for code:
https://github.com/google/material-design-icons/blob/master/iconfont/codepoints

Is there anything similar for Framework7-icons?

Regards,

**Comment by The-Don-Himself** [comment]
I have also ran into this while testing on an Android 4.2.2 device, which is still extremely popular this part of the world.

The above suggestion should help correct things as I have tested it with material icons and the workaround works, curiously though, I have also tested font-awesome and it works out of the box without a need for a workaround so perhaps there is something to be borrowed from there.

**Comment by scriptPilot** [comment]
@The-Don-Himself - Font Awesome does not use ligatures, that's the point. **App Framework** does the replacement for Material automatically - https://github.com/scriptPilot/app-framework

**Comment by The-Don-Himself** [comment]
@scriptPilot you are correct about the ligatures, I had also seen an SO answer that said that was the problem although I din't know that Font-Awesome dint use them, thanks for enlightening me.

Your App Framework seems really awesome, actually wish I found it sooner and will definitely look more into it. Just, starred the project!
